lib/phoenix_live_view/igniter/upgrade_to_1_1.ex

```diff
@@ -3,8 +3,7 @@ defmodule Phoenix.LiveView.Igniter.UpgradeTo1_1 do
 
   def run(igniter, _opts) do
     igniter
-    |> Igniter.Project.Deps.add_dep({:phoenix_live_view, "~> 1.1"})
```

Comment:
This will be changed as part of the wrapping upgrade logic, your upgrader doesn't need to apply it.